docs/setup/settings.asciidoc

```diff
@@ -52,67 +96,31 @@ The following example shows a valid regionmap configuration.
 
-`elasticsearch.username:` and `elasticsearch.password:`:: If your Elasticsearch is protected with basic authentication,
-these settings provide the username and password that the Kibana server uses to perform maintenance on the Kibana index at
-startup. Your Kibana users still need to authenticate with Elasticsearch, which is proxied through the Kibana server.
+`server.basePath:`:: Enables you to specify a path to mount Kibana at if you are running behind a proxy. This only affects
+ the URLs generated by Kibana, your proxy is expected to remove the basePath value before forwarding requests
+ to Kibana. This setting cannot end in a slash (`/`).
+`server.customResponseHeaders:`:: *Default: `{}`* Header names and values to send on all responses to the client from the Kibana server.
+`server.defaultRoute:`:: *Default: "/app/kibana"* This setting specifies the default route when opening Kibana. You can use this setting to modify the landing page when opening Kibana.
+`server.host:`:: *Default: "localhost"* This setting specifies the host of the back end server.
+dashboards. Kibana creates a new index if the index doesn’t already exist.
+`server.maxPayloadBytes:`:: *Default: 1048576* The maximum payload size in bytes for incoming server requests.
+`server.name:`:: *Default: "your-hostname"* A human-readable display name that identifies this Kibana instance.
+`server.port:`:: *Default: 5601* Kibana is served by a back end server. This setting specifies the port to use.
 `server.ssl.enabled`:: *Default: "false"* Enables SSL for outgoing requests from the Kibana server to the browser. When set to `true`, `server.ssl.certificate` and `server.ssl.key` are required
 `server.ssl.certificate:` and `server.ssl.key:`:: Paths to the PEM-format SSL certificate and SSL key files, respectively.
 `server.ssl.keyPassphrase`:: The passphrase that will be used to decrypt the private key. This value is optional as the key may not be encrypted.
 `server.ssl.redirectHttpFromPort` :: Kibana will bind to this port and redirect all http requests to https over the port configured as `server.port`.
 `server.ssl.certificateAuthorities`:: List of paths to PEM encoded certificate files that should be trusted.
 `server.ssl.supportedProtocols`:: *Default: TLSv1, TLSv1.1, TLSv1.2*  Supported protocols with versions. Valid protocols: `TLSv1`, `TLSv1.1`, `TLSv1.2`
 `server.ssl.cipherSuites`:: *Default: ECDHE-RSA-AES128-GCM-SHA256, ECDHE-ECDSA-AES128-GCM-SHA256, ECDHE-RSA-AES256-GCM-SHA384, ECDHE-ECDSA-AES256-GCM-SHA384, DHE-RSA-AES128-GCM-SHA256, ECDHE-RSA-AES128-SHA256, DHE-RSA-AES128-SHA256, ECDHE-RSA-AES256-SHA384, DHE-RSA-AES256-SHA384, ECDHE-RSA-AES256-SHA256, DHE-RSA-AES256-SHA256, HIGH,!aNULL, !eNULL, !EXPORT, !DES, !RC4, !MD5, !PSK, !SRP, !CAMELLIA*. Details on the format, and the valid options, are available via the [OpenSSL cipher list format documentation](https://www.openssl.org/docs/man1.0.2/apps/ciphers.html#CIPHER-LIST-FORMAT)
```

Comment:
I know this isn't part of your changes, but all these commas are silly. Simply `Details on the format and the valid options are available via the ...`